extensions/adobe/experience/adcloud/adcloudremarketingprofile.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/adcloudremarketingprofile",
+  "title": "Adobe AdCloud Search Profile Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Adobe AdCloud Search extension to Profile",
+  "definitions": {
+    "adcloudremarketingprofile": {
+      "properties": {
+        "xdm:syncedRemarketingAudiences": {
+          "title": "Synchronized segments",
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/experience/adcloud/syncedremarketingaudience"
+          },
+          "description": "List of Search retargeting entities"
```

Comment:
This is description is not helping me understand the actual way to use this, can it be improved please?